node-tests/unit/utilities/assets/unknown-test.js

```diff
@@ -12,5 +12,23 @@ describe('UnknownAdapter', function() {
       unknown = new UnknownAdapter();
       expect(unknown.upload()).to.be.rejected;
     });
-  })
+  });
+  describe('#activate', function() {
+    it('rejects to print out an error message', function(){
+      unknown = new UnknownAdapter();
```

Comment:
All these tests are basically the same. We can use a function to create the `expect(unknown.<method>)` calls. Also the initialization of the unknown adapter could be moved into a beforeEach function.